microservices/nris_api/postgres/Dockerfile.dev

```diff
@@ -0,0 +1,7 @@
+FROM postgres:9.6
+
+# Copy only migration scripts for creation
+RUN echo "CREATE USER postgres" >> /docker-entrypoint-initdb.d/01-create-user.sql
+
+# Set the working directory to container entrypoint
+WORKDIR /docker-entrypoint-initdb.d
```

Comment:
Is this file complete? What's the reason for setting WORKDIR as the last line of the file?

Reply:
I think setting workdir also sets the context for any CMD or ENTRYPOINT commands passed into the container at runtime. E.g. `docker run container_name relative_file_path` , when the container executes its entrypoint script it will look at the `relative_file_path` command passed to it in the workdir directory.

Reply:
It may work without the WORKDIR directive as well. I think I had errors and then I ended up putting that in, but that was a while back and im not certain if this is needed or not anymore